### PR TITLE
Update ec2_vpc_net docs

### PR DESCRIPTION
**Pull request by prasadkatti**
- update the docs to use complex return type

##### SUMMARY
Update ec2_vpc_net docs

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME
`ec2_vpc_net`

##### ANSIBLE VERSION

```
devel
```

### CODE REVIEW

**Comment by willthames** [comment]
@prasadkatti, FYI, it's always worth checking https://ansible.sivel.net/pr/byfile.html (not that I always remember to do so)

In this case I'm really keen to get #33105 in first - but perhaps @s-hertel could rebase on yours if we merge this in.

File: lib/ansible/modules/cloud/amazon/ec2_vpc_net.py [hunk 1]
```diff
@@ -311,7 +319,11 @@ def main():
                 e_msg = boto_exception(e)
                 module.fail_json(msg=e_msg)
 
-        module.exit_json(changed=changed, vpc=get_vpc_values(vpc_obj))
+        vpc = get_vpc_values(vpc_obj)
+        if vpc is not None:
```
Comment (s-hertel) [inline lib/ansible/modules/cloud/amazon/ec2_vpc_net.py]: Is vpc able to be None if state is present?

**Comment by s-hertel** [comment]
@prasadkatti @willthames Thanks. I don't mind rebasing, it will be an easy one.

**Comment by prasadkatti** [comment]
ready_for_review

**Review by s-hertel — APPROVED** [review]

**Comment by s-hertel** [comment]
Thanks @prasadkatti!

**Pull request merged**